common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -129,6 +129,22 @@ is_kilrogg_trigger = {
 	exists = character:10005 # Kilrogg
```

Comment:
I guess you tweaked them too much so they look like different species. Especially Urukal.
Their heads became too tall. Chin became less expressive.

Reply:
I mean, I get Urukal's head being tall, and I lowered the forehead height of Griselda, but I think Dal'rend and Maim look pretty close to the other orcs, especially compared to the way Blackhand looks. Can you explain what you mean?

Reply:
> I lowered Urukal's head and the forehead height of Griselda in the commit I just put out, but I think Dal'rend and Maim look pretty close to the other orcs, especially compared to the way Blackhand looks. Can you explain what you mean?

After the second look, they look better than other orcs.